mern/server/models/question_answer.js

```diff
@@ -1,27 +1,29 @@
-import mongoose from "mongoose"
+import mongoose from "mongoose";
 
 const Schema = mongoose.Schema;
 
 const QASchema = new Schema({
-    id: { 
-        type: String,
-        require: true,
-        unique: true
+  id: {
```

Comment:
Are you changing the format here or are they still the same? (I can't seem to find the difference)😅